eng/source-build-release-announcement.md

```diff
@@ -0,0 +1,9 @@
+<!-- This file is a template for a GitHub Discussion post.  -->
+<!-- The line prefixed by 'Title:' will be submitted as the title of the discussion, and the rest of the file will be submitted as the body. -->
+Title: .NET $DOTNET_MAJOR_MINOR_VERSION $DATE Update - .NET $RUNTIME_VERSION and SDK $SDK_VERSION
+
+[Release Notes]($RELEASE_NOTES_URL) | [Tag]($TAG_URL)
+
+Please use the [$TAG tag]($TAG_URL) to source-build .NET version $RUNTIME_VERSION / $SDK_VERSION.
+
+@distro-maintainers
```

Comment:
I'd expect this to need @dotnet/distro-maintainers - does this work because it's the same org?

Reply:
Good catch - I wasn't sure if we needed it or not but I just tried mentioning it in this comment and it looks like it does need the org prefix.

Reply:
I updated it. Also, if the mention does not work we can manually fix it, at least the first time.